### PR TITLE
require boot dependencies from manageiq app

### PR DESCRIPTION
**Pull request by durandom**
Dont duplicate dependencies  but require from manageiq test app

fixes: https://github.com/ManageIQ/manageiq-providers-amazon/issues/1
~~requires: https://github.com/ManageIQ/manageiq/pull/9507~~
requires: https://github.com/ManageIQ/manageiq/pull/9534

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -14,58 +12,5 @@ end
...
-gem "hamlit",                         "~>2.0.0",   :require => false
-gem "image-inspector-client",  "~>1.0.2",           :require => false
-gem "kubeclient",              "=1.1.3",            :require => false
-gem "ansible_tower_client",           "~>0.3.0",   :require => false
-gem "rubywbem",            :require => false, :git => "git://github.com/ManageIQ/rubywbem.git", :branch => "rubywbem_0_1_0"
-gem "net_app_manageability",          ">=0.1.0",   :require => false
-gem "coveralls",                    :require => false
-gem "hawkular-client",         "=2.0.0",            :require => false
+# Load Gemfile with dependencies from manageiq
+eval_gemfile(File.expand_path("spec/manageiq/Gemfile", __dir__))
```
Comment (chessbyte) [inline Gemfile]: should this be `Gemfile.boot`?

Comment (durandom) [inline Gemfile]: I tried this before, but I think those are not really boot dependencies but those to make the tests run. See [here for a bit more discussion](https://github.com/ManageIQ/manageiq/pull/9507#issuecomment-229584194)

**Comment by durandom** [comment]
requires https://github.com/ManageIQ/manageiq/pull/9622 to be merged

**Comment by durandom** [comment]
@Fryguy its :green_apple: merge?

**Comment by bdunne** [comment]
:+1: LGTM

**Pull request merged**